src/style.css

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.margin-form {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.input {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  width: 100%;
+  height: 70px;
+  line-height: 22px;
+  padding-left: 62px;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background: #fff;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: absolute;
+  background-position-x: 26px;
+  background-position-y: 25px;
+  background-size: 19px 19px;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  font-weight: bold;
+  border-radius: 3px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: absolute;
+  background-position-x: 26px;
+  background-position-y: 25px;
+  background-size: 19px 19px;

```

Comment:
u r repeating your background for 4 times, it is not ok. Use tag `img` or pseudoelement for background image

Reply:
haven't fixed